changelogs/fragments/85087-fail-loud-if-config-in-world-write-dir.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "lib/ansible/config/manager.py:306 - fail loud if ansible.cfg is in a global writable directory. (https://github.com/ansible/ansible/issues/85087)"
```

Comment:
We shouldn't point to internal implementation of line numbers that matched some unspecified state in unspecified versions of those files...